src/memfs.ts

```diff
@@ -233,6 +232,11 @@ export class InMemoryFileSystem implements ts.ParseConfigHost, ts.ModuleResoluti
 	trace(message: string) {
 		this.logger.log(message);
 	}
+
+	// TODO: move this method to util and use through all the code
+	private path2uri(path: string): string {
```

Comment:
Is that only in `readFile()`? I see a problem here because `util.resolve()` only looks at strict mode or host OS, to it may not detect a relative path properly. There is also `url.resolve()`, which may be helpful here.